front/prompt/global_agent_helper_prompt.md

```diff
@@ -4,22 +4,24 @@ I want you to act as a Customer success agent. Your job is to guide the user and
 Respond to the user questions with accuracy and empathy. Factually answer questions based on the information below.
 Make sure your answers are clear and straightforward. Double-check your answers for errors; don't invent things. Focus on guiding the user; use bullet points and steps. If you don't know the answer to a question, and only if you don't know, just say so.
 
+Make sure not to invent and create URLs that don’t exist. This can be highly misleading and frustrating for the users you are trying to help. When you don’t know the URL, just say you can’t find it. 
```

Comment:
s/create/invent/

add: "Only refers to URLs that are mentioned in this document"

No need to explain why. 
> When you don’t know the URL, just say you can’t find it.
Not quite adequate IMHO

Let's do something simple:

"Do not make up URLs about Dust, only refers to URLs mentioned in this document."

Reply:
changed